_rules/first-focusable-elements-are-links-to-content-e53727.md

```diff
@@ -1,159 +1,356 @@
+	</body>
+</html>
+```
+
+#### Failed Example 2
+
+List of links when activated do not move the focus to a section of content.
+
+```html
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<title></title>
+	</head>
+	<body>
+		<ul>
+			<li><a href="#" aria-label="Skip to navigation"></a></li>
+			<li><a href="/anotherpage" aria-label="Skip to search"></a></li>
+			<li><a href="javascript:void(0);" aria-label="Skip to main content"></a></li>
+		</ul>
+		<!-- Repeated content -->
+		<nav id="navigation">Navigation section</nav>
+		<form id="search" role="search">Search section</form>
+		<main id="main">Main content</main>
+	</body>
+</html>
+```
+
+#### Failed Example 3
+
+List of links do not move focus to a section of content when activated due to invalid id references.
+
+```html
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<title></title>
+	</head>
+	<body>
+		<ul>
+			<li><a href="#navigation" aria-label="Skip to navigation"></a></li>
+			<li><a href="#search" aria-label="Skip to search"></a></li>
+			<li><a href="#main" aria-label="Skip to main content"></a></li>
+		</ul>
+		<!-- Repeated content -->
+		<nav>Navigation section</nav>
+		<form role="search">Search section</form>
+		<main>Main content</main>
+	</body>
+</html>
+```
+
+#### Failed Example 4
+
+List of links segment elements are not the first focusable elements in the document.
+
+```html
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<title></title>
+	</head>
+	<body>
+		<a href="/anotherpage">Go to another page</a>
+		<ul>
+			<li><a href="#">Skip to navigation</a></li>
+			<li><a href="/anotherpage">Skip to search</a></li>
+			<li><a href="javascript:void(0);">Skip to main content</a></li>
+		</ul>
+		<!-- Repeated content -->
+		<nav id="navigation">Navigation section</nav>
+		<form id="search" role="search">Search section</form>
+		<main id="main">Main content</main>
+	</body>
+</html>
+```
+
+#### Failed Example 5
+
+List of links are not visible when focus is applied.
+
+```html
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<title></title>
+	</head>
+	<body>
+		<ul style="display:none;" id="skiplinks">
+			<li><a href="#navigation">Skip to navigation</a></li>
+			<li><a href="#search">Skip to search</a></li>
+			<li><a href="#main">Skip to main content</a></li>
+		</ul>
+		<!-- Repeated content -->
+		<nav id="navigation">Navigation section</nav>
+		<form id="search" role="search">Search section</form>
+		<main id="main">Main content</main>
+	</body>
+</html>
+```
+
+#### Failed Example 6
+
+List of links does not have an accessible name.
+
+```html
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<title></title>
+	</head>
+	<body>
+		<ul>
+			<li><a href="#navigation"></a></li>
+			<li><a href="#search"></a></li>
+			<li><a href="#main"></a></li>
+		</ul>
+		<!-- Repeated content -->
+		<nav id="navigation">Navigation section</nav>
+		<form id="search" role="search">Search section</form>
+		<main id="main">Main content</main>
+	</body>
+</html>
+```
+
```

Comment:
Add a failed example where not all sections of content are covered (_i.e._ failing Expectation 2).
Add example where one section of content is covered by several links.